servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespace/README.md

```diff
@@ -1,183 +1,89 @@
-    ```
-    > We should specify `spring.profiles.active=mi` to run the Spring Boot application.
-    For App Service, please add a configuration entry for this.
+# Login to azure-cli
+az login
+az account set --subscription <your-subscription-id>
 
-##### Redeploy Application
+# 
+terraform init
+# it may take a few minutes
+terraform apply
+```
 
-If you update the `spring.cloud.azure.credential.managed-identity-client-id`
-property after deploying the app, or update the role assignment for
-services, please try to redeploy the app again.
 
-> You can follow
-> [Deploy a Spring Boot JAR file to Azure App Service][deploy-spring-boot-application-to-app-service]
-> to deploy this application to App Service
+## Run locally
 
-## Examples
+Open your terminal, and input the command below.
 
-1. Run the `mvn spring-boot:run` in the root of the code sample to get the app running.
+```shell
+mvn clean spring-boot:run
+```
 
-2. Update queue name in 
-   [QueueReceiveController.java][queue-receive-controller] and
-   [QueueSendController.java][queue-send-controller],
-   and update topic name and subscription in
-   [TopicReceiveController.java][topic-receive-controller] and
-   [TopicSendController.java][topic-send-controller].   
-    
-3. Run the `mvn spring-boot:run` in the root of the code sample to get the app running.
 
-4. Send a POST request to service bus queue
+## Run on App Service
+You can also run the application on [Azure App Services.](https://azure.microsoft.com/services/app-service/)
 
-        $ curl -X POST http://localhost:8080/queues?message=hello
+### Deploy With Maven
+1. Fill in the value in the pom.xml file.
 
-5. Verify in your app’s logs that a similar message was posted:
+```shell
+<subscriptionId>${your-subscriptionId}</subscriptionId>
+<resourceGroup>${your-resourceGroup}</resourceGroup>
+<appName>${your-appName}</appName>
+```
 
-        New message received: 'hello'
-        Message 'hello' successfully checkpointed
-6. Send a POST request to service bus topic
+2. Open you terminal and run
+```shell
+mvn package azure-webapp:deploy
+```
 
-        $ curl -X POST http://localhost:8080/topics?message=hello
+### Deploy With Intellj plugin
+If you want to upload with Intellj plugins you can check this [doc](https://docs.microsoft.com/en-us/azure/developer/java/toolkit-for-intellij/create-hello-world-web-app#deploying-web-app-to-azure) for guides.
 
-7. Verify in your app’s logs that a similar message was posted:
-
-        New message received: 'hello'
-        Message 'hello' successfully checkpointed
-
-8. Delete the resources on [Azure Portal][azure-portal] to avoid unexpected charges.
 
-## Enhancement
-### Set Service Bus message headers
-The following table illustrates how Spring message headers are mapped to Service Bus message headers and properties.
-When creat a message, developers can specify the header or property of a Service Bus message by below constants.
+## Clean up Resources
```

Comment:
maybe add some descrption in this section like

" do not forget to clean up resources once you finished the demo, or it will continue to charge you"